projects/igniteui-angular/src/lib/core/utils.ts

```diff
@@ -33,7 +33,7 @@ export const mkenum = <T extends { [index: string]: U }, U extends string>(x: T)
  *
  * @hidden @internal
  */
-export const getResizeObserver = () => window.ResizeObserver || Polyfill;
+export const getResizeObserver = () => window?.ResizeObserver || Polyfill;
```

Comment:
@rkaraivanov I left the native implementation change in (as it seems like a good idea) w/ one small nullish check to avoid possible SSR issues.